### PR TITLE
Add chain upgrade on none capset

### PR DESCRIPTION
**Pull request by shellyyang1989**
cc @jiajliu @jianlinliu @JianLi-RH

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.11.yaml [hunk 1]
```diff
@@ -55,6 +55,42 @@ resources:
...
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.13 and @aws-ipi and not @fips
+        and not @storage and not @console'
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.13 and @aws-ipi and not @fips
+        and not @storage and not @console'
+    post:
+    - chain: cucushift-installer-rehearse-aws-ipi-deprovision
+    pre:
+    - chain: ipi-conf-aws
+    - chain: ipi-install
+    - ref: enable-qe-catalogsource
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.11.yaml:75]: I guess we can directly call `chain: cucushift-installer-rehearse-aws-ipi-provision` here?

Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.11.yaml:75]: yeah, agree. We can even use the workflow directly.

**Pull request closed without merging**